var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -25,25 +25,27 @@
 from spack import *
 
 
-class Libint(Package):
+class Libint(AutotoolsPackage):
     """Libint is a high-performance library for computing
-    Gaussian integrals in quantum mechanics."""
+    Gaussian integrals in quantum mechanics.
+    """
 
     homepage = "https://github.com/evaleev/libint"
-    url      = "https://github.com/evaleev/libint/archive/v2.1.0.tar.gz"
+    url = "https://github.com/evaleev/libint/archive/v2.1.0.tar.gz"
 
+    version('2.2.0', 'da37dab862fb0b97a7ed7d007695ef47')
     version('2.1.0', 'd0dcb985fe32ddebc78fe571ce37e2d6')
     version('1.1.6', '990f67b55f49ecc18f32c58da9240684')
     version('1.1.5', '379b7d0718ff398715d6898807adf628')
 
     # Build dependencies
     depends_on('autoconf@2.52:', type='build')
-    depends_on('automake',       type='build')
-    depends_on('libtool',        type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
```

Comment:
It seems I have E241 and E242 turned on on `autopep8`. I can't say that I don't agree with them, but if this bothers people I can revert the changes.